snowflake-api/Cargo.toml

```diff
@@ -32,14 +33,16 @@ reqwest-middleware = "0.2"
 reqwest-retry = "0.3"
 serde = { version = "1", features = ["derive"] }
 serde_json = "1"
-snowflake-jwt = { version = "0.3.0", optional = true }
 thiserror = "1"
 url = "2"
 uuid = { version = "1", features = ["v4"] }
+snowflake-jwt = { version = "0.3.0", optional = true }
+glob = { version = "0.3.1", optional = true }
+tokio = { version = "1.35", optional = true, features = ["macros", "rt-multi-thread"] }
```

Comment:
It's bad practice to fix minor versions if you absolutely don't have to. Library users would be limited by this even if they have legitimate reasons to use specific version.